pandas/compat/pyarrow.py

```diff
@@ -18,8 +18,9 @@
     pa_version_under19p0 = _palv < Version("19.0.0")
     pa_version_under20p0 = _palv < Version("20.0.0")
     pa_version_under21p0 = _palv < Version("21.0.0")
+    pa_version_under22p0 = _palv < Version("22.0.0")
     HAS_PYARROW = _palv >= Version(PYARROW_MIN_VERSION)
-except ImportError:
+except (ImportError,AttributeError):
```

Comment:
the linter is going to wanta  space after the comma

Reply:
Ah! Fixed. Thank you!